Tests/Unit/Fixtures/Hooks/TestingHookImplementor2.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks;
+
+use OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks\TestingHookImplementor;
+
+/**
+ * Second valid test interface implementation to use with the HookProviderTest
+ *
+ * Will be accepted as a second hooked-in class
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+class TestingHookImplementor2 extends TestingHookImplementor
+{
+    /**
+     * @var int
+     */
+    public static $wasCalled = 0;
```

Comment:
Doesn't it work to have this field in the parent class only (if it is accessed using `static::$wasCalled`)?

Reply:
No, doesn't work. To get distinct properties per class the variable must be repeated.